src/ContentValidationListener.php

```diff
@@ -237,12 +238,15 @@ public function onRoute(MvcEvent $e)
 
         $e->setParam('ZF\ContentValidation\InputFilter', $inputFilter);
 
+        $event = clone $e;
```

Comment:
We don't actually need to clone it. In v2, if you pass an event in, it sets the name directly on the instance. As such, you can just call `$e->setName()`.